docs/website/docs/hub/getting-started/installation.md

```diff
@@ -126,40 +128,45 @@ C:\> .\venv\Scripts\activate
 `dlthub` will be automatically installed with workspace extra:
 
 ```sh
-# install the newest dlt version or upgrade the existing version to the newest one
+# install the newest dlt[workspace] version or upgrade the existing version to the newest one
 uv pip install -U "dlt[workspace]"
 ```
 
-Please install a valid license before proceeding, as described under [licensing](#licensing).
+Please install a valid license before proceeding, as described under [licensing](#self-licensing).
 
-## Licensing
 
-Once you have a valid license, you can make it available to dlthub using one of the following methods:
 
-1. **Environment variable**: set the license key as an environment variable:
+## Self-licensing
 
-```sh
-export RUNTIME__LICENSE="eyJhbGciOiJSUz...vKSjbEc==="
-```
 
-2. **Secrets file**: add the license key to a `secrets.toml` file. You can use either the project-level `secrets.toml` (located in `./.dlt/secrets.toml`) or the global one (located in `~/.dlt/secrets.toml`):
+You can self-issue an anonymous 30-day trial license to test dlthub features and use it for development, eductation, and ci ops. The self-issued licenses are not granted for "Production Use" and cannot be transfered to other machines/workspaces. 
```

Comment:
```suggestion
You can self-issue an anonymous 30-day trial license to test dlthub features and use it for development, education, and CI ops. The self-issued licenses are not granted for use in production and cannot be transferred to other machines/workspaces. 
```
Could you also mention more explicitly, that the license is bound to the machine it was issued on?